examples/fluids/qfunctions/advection.h

```diff
@@ -11,8 +11,8 @@
 #ifndef advection_h
 #define advection_h
 
+#include <ceed/types.h>
```

Comment:
The previous code will (should) work fine here as `<ceed.h>` is still ignored in JiT, but this header has less stuff in it, so I switched all of these `#include <ceed.h>` over to the new `#include <ceed/types.h>`